scripts/repochecker

```diff
@@ -4,7 +4,7 @@
 # used for the openSUSE/SLE Cloud packages to check the dependencies for the
 # different repositories. Use it for instance with:
 #
-# REPCHECK_DIST_VERSION=13.2 REPCHECK_DIST_NAME=openSUSE \
+# REPCHECK_DIST_VERSION=15.0 REPCHECK_DIST_NAME=openSUSE \
 #     REPCHECK_PROJECT="Cloud:OpenStack:Kilo:Staging" repochecker
```

Comment:
You might as well update this comment also. Kilo is just a tiny bit oldish :wink: